File: test/Video/test06.php
#!/usr/bin/php -c.
<?php

$srcdir = $argv[1];

$m = new SWFMovie(7);

$stream = new SWFVideoStream($srcdir . "/../Media/video01.flv");

$stream->setFrameMode(SWF_VIDEOSTREAM_MODE_MANUAL);
$stream->setDimension(200, 200);

$m->add($stream);

$off = 0;
for($i = 0; $i < 50; $i++)
{
	if ( $i == 30 )
		$off = $stream->seek(50, SEEK_SET);

	if ( $i == 40 )
		$stream->seek($off, SEEK_SET);
	
	$stream->nextFrame();
	$m->nextFrame();
}

$m->save("test06.swf");

?>

File: test/Video/test06.py
#!/usr/bin/python
from ming import *
import sys
import os

srcdir=sys.argv[1]

Ming_useSWFVersion(7);
m = SWFMovie();

stream = SWFVideoStream(srcdir + "/../Media/video01.flv");
stream.setDimension(200, 200);
stream.setFrameMode(SWFVIDEOSTREAM_MODE_MANUAL);

m.add(stream);

for i in range(50):
	if i == 30 : off = stream.seek(50, os.SEEK_SET);
	if i == 40 : stream.seek(off, os.SEEK_SET);
	stream.nextFrame();
	m.nextFrame();

m.save("test06.swf");
